pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/DeclarativeLinterCommandTest.java

```diff
@@ -75,6 +79,18 @@ public void invalidJenkinsfile() throws Exception {
         assertThat(result.stdout(), containsString("Not a valid section definition: 'agent'. Some extra configuration is required"));
     }
 
+    @Test
+    public void invalidUser() throws Exception {
+        File testPath = writeJenkinsfileToTmpFile("simplePipeline");
+
+        j.jenkins.setAuthorizationStrategy(new GlobalMatrixAuthorizationStrategy());
```

Comment:
There is no need to add matrix-auth as a dependency, when you have access to MockAuthorisationStrategy
Example:
```java
j.jenkins.setSecurityRealm(j.createDummySecurityRealm());
j.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy()
                                                   .grant(Jenkins.ADMINISTER).everywhere().to("bob")
                                                   .grant(Jenkins.READ, 
                                                             Item.READ, 
                                                             Item.EXTENDED_READ).everywhere().to("alice"));
```